frontend-react/.storybook/preview.js

```diff
@@ -0,0 +1,59 @@
+import { initializeWorker, mswDecorator } from "msw-storybook-addon";
+import { CacheProvider } from "rest-hooks";
+import { BrowserRouter } from "react-router-dom";
+import { HelmetProvider } from "react-helmet-async";
+import MockDate from "mockdate";
+
+import "../src/content/generated/global.out.css";
+
+// mock all dates in stories to make sure we don't run into date-related inconsistencies
+MockDate.set("2023-01-01");
```

Comment:
This'd be especially useful if we start using snapshots with Chromatic.

Reply:
I'm curious how this will interact with faker.js if we end up using it but something to worry about later.